resilience4j-metrics/src/main/java/io/github/resilience4j/metrics/Timer.java

```diff
@@ -17,9 +17,11 @@
 public interface Timer {
 
     /**
-     * Starts the Timer
+     * Creates a Timer context and starts the timer
+     *
+     * @return the Timer context
      */
-    Context time();
+    Context context();
```

Comment:
I see too manny things called “context”. 
It is vary hard to distinguish them if you see this code first time.
I’d suggest to think about other names.